chain-signatures/node/src/protocol/signature.rs

```diff
@@ -551,7 +553,25 @@ impl SignatureManager {
                 .max_gas()
                 .retry_exponential(10, 5)
                 .transact()
-                .await?;
+                .await
+            {
+                Ok(response) => response,
+                Err(err) => {
+                    tracing::error!(%receipt_id, error = ?err, "Failed to publish transaction");
+                    break;
```

Comment:
We should be doing `continue` instead of `break` since being unable to publish one signature shouldn't be affecting another signature from being published.